### ISSUE TITLE
Missing Spanish Holiday

### ISSUE THREAD

**Issue by jairsan** [ryanss/python-holidays#70]
The national day of Spain seems to be missing from the holiday list. It is held every year on October 12.

```
>>> import holidays
>>> hd = holidays.ES()
>>> hd.get('2016-10-12')
```

More information can be found on the Wikipedia page: https://en.wikipedia.org/wiki/Fiesta_Nacional_de_Espa%C3%B1a

**Comment by Pablo-Merino** [comment]
I added this as a pull request in #74. Just waiting for the merge! In the meanwhile, you can use my fork

**Comment by dr-prodigy** [comment]
Merged in **1452cc3**
